make/package

```diff
@@ -25,7 +25,7 @@ mkdir -p ./output/packaging
 	cd ./output
 	(
 		cd ./packaging
-		curl -sS -L -O --fail http://d6r77u77i8pq3.cloudfront.net/releases/${traveling_ruby}
+		curl -sS -L -O --fail https://storage.googleapis.com/pelerinul/${traveling_ruby}
```

Comment:
I guess this is your account? Is that going to last ~forever?
It looks like upstream has a `traveling-ruby-20150715-2.2.2-linux-x86_64.tar.gz`; it's a bit older, but looks like the same version of Ruby.  Is that sufficient?